setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
That sounds awful :-(
Good thing us, as soon as Travis support lands, it will be easier to test different installation scenarios.

Also, note that you can use the extras_require kwarg to specify optional dependencies, so in fact users wouldnt have to install those separately. See http://setuptools.readthedocs.io/en/latest/setuptools.html#id14